packages/utilities/src/utilities/markdownToHtml/markdownToHtml.js

```diff
@@ -0,0 +1,71 @@
+import { settings } from 'carbon-components';
+const { prefix } = settings;
+
+const _htmlTagRegex = /<.*?>/g;
+const _doubleSpaceRegex = /\s{2,}/g;
+const _italicRegex = /[_*](.*?)[_*]/g;
+const _boldRegex = /[_*]{2}(.*?)[_*]{2}/g;
+
+/**
+ * Removes any html tags from a string and keeps inner text if any
+ *
+ * @param {string} str String to be checked for html tags
+ * @returns {string} String with html tags stripped out
+ */
+const _removeHtmlTags = str => str.replace(_htmlTagRegex, '');
+
+/**
+ * Checks for double spaces between words in a string and replaces for single spaces
+ *
+ * @param {string} str String to be checked for double spaces
+ * @returns {string} String with html double spaces fixed
+ */
+const _fixDoubleSpaces = str => str.replace(_doubleSpaceRegex, ' ');
+
+/**
+ * Converts some markdown syntaxes into html
+ * It's not a full markdown-to-html converter
+ * It currently supports two syntaxes: Bold and Italic
+ * Bold: Double asterisk (**) or double underscore (__).
+ * Bold examples: **Lorem ipsum** __dolor__
+ * Italic: Single asterisk (*) or single underscore (_)
+ * Italic examples: _Lorem ipsum_ *dolor*
+ *
```

Comment:
@emerson-pereira just checked out the JSDoc output, the formatting seems a bit hard to read:
https://deploy-preview-872--ibmdotcom-utilities.netlify.com/global.html#markdownToHtml

I believe you can add in HTML formatting so you can retain formatting in descriptions, might be worth trying:

```
/**
 * Converts some markdown syntaxes into html
 * <p>It's not a full markdown-to-html converter</p>
 * <p>It currently supports two syntaxes: <strong>Bold</strong> and <em>Italic</em></p>
 * <ul>
 * <li>Bold: Double asterisk (**) or double underscore (__).</li>
 * <li>Bold examples: **Lorem ipsum** __dolor__</li>
 * <li>Italic: Single asterisk (*) or single underscore (_)</li>
 * <li>Italic examples: _Lorem ipsum_ *dolor*</li>
 * </ul>
 */
```

Reply:
Good to know! Implemented that, let's see...